get-telemetry.py

```diff
@@ -0,0 +1,101 @@
+import os
+import csv
+import requests
+import urllib
+from datetime import datetime, timedelta
+
+MAX_REQUEST_SIZE_HOURS = 48
+SECONDS_IN_HOUR = 3600.0
+VOLTUS_API_URL = os.getenv("VOLTUS_API_URL", "https://api.voltus.co")
+VOLTUS_API_KEY = os.getenv("VOLTUS_API_KEY", "secret")
```

Comment:
`secret` just isn't gonna work, is it? I think we're better off just aborting if the user doesn't set it. 

```suggestion
VOLTUS_API_KEY = os.getenv("VOLTUS_API_KEY")
```